blockchain/blockdao/filedao.go

```diff
@@ -69,7 +74,7 @@ func NewFileDAO(compressBlock bool, cfg config.DB) (FileDAO, error) {
 	if err != nil {
 		return nil, err
 	}
-	return createFileDAO(legacyFd)
+	return createFileDAO(legacyFd, nil)
```

Comment:
still use legacy file for now
next PR will switch knob to use combination of legacy and new file